modules/installation-mirror-repository.adoc

```diff
@@ -76,11 +113,84 @@ imageContentSources:
   - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
   source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
 ...
+endif::install[]
+ifdef::update[]
+To use the new mirrored repository for upgrades, use the following to create an ImageContentSourcePolicy:
+
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: example
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
+    source: quay.io/openshift-release-dev/ocp-release
+  - mirrors:
+    - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
+    source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+...
+endif::update[]
+----
+
+. Record the
+ifdef::install[]
+`imageContentSources`
+endif::install[]
+ifdef::update[]
+`ImageContentSourcePolicy`
+endif::update[]
+section from the output of the previous
+command. This information is required
+ifdef::install[]
+during {product-title} installation.
+endif::install[]
+ifdef::update[]
+when you update your {product-title} cluster.
+endif::update[]
+
+. Build the signature for the content that you mirrored and verify that it is
+signed by an official Red Hat key:
+.. Set the digest for the version to install to:
++
+----
+$ DIGEST="$(oc adm release info quay.io/openshift-release-dev/ocp-release:<release_version> | sed -n 's/Pull From: //p')" <1>
+----
+<1> For `<release_version>`, specify the version number of {product-title} to
+that you mirrored content for.
```

Comment:
You want to verify the signature before going through the trouble of mirroring, and then use a by-digest pullspec for the mirror command instead of the `<release_version>` tag.